infra/discovery/src/listener/listener.js

```diff
@@ -235,4 +245,12 @@ logger.info(
 )
 
 setNetwork(config.network)
-main()
+try {
+  main().catch(err => {
```

Comment:
are both try/catch statement and .catch handler necessary ? Isn't it redundant ?

Reply:
I have no idea what happened here... thanks